plugins/plugin-chart-echarts/src/BoxPlot/controlPanel.ts

```diff
@@ -0,0 +1,114 @@
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { t, validateNonEmpty } from '@superset-ui/core';
+import { D3_FORMAT_OPTIONS, formatSelectOptions } from '@superset-ui/chart-controls';
+
+export default {
+  controlPanelSections: [
+    {
+      label: t('Query'),
+      expanded: true,
+      controlSetRows: [
+        ['metrics'],
+        ['adhoc_filters'],
+        ['groupby'],
+        ['columns'],
+        ['limit'],
+        [
+          {
+            name: 'whiskerOptions',
+            config: {
+              type: 'SelectControl',
+              freeForm: true,
+              label: t('Whisker/outlier options'),
+              default: 'Tukey',
+              description: t('Determines how whiskers and outliers are calculated.'),
+              choices: formatSelectOptions([
+                'Tukey',
+                'Min/max (no outliers)',
+                '2/98 percentiles',
+                '9/91 percentiles',
+              ]),
+            },
+          },
+        ],
+      ],
+    },
+    {
+      label: t('Chart Options'),
+      expanded: true,
+      controlSetRows: [
+        ['color_scheme'],
+        [
+          {
+            name: 'x_ticks_layout',
+            config: {
+              type: 'SelectControl',
+              label: t('X Tick Layout'),
+              choices: formatSelectOptions(['auto', 'flat', '45°', '90°', 'staggered']),
+              default: 'auto',
+              clearable: false,
+              renderTrigger: true,
+              description: t('The way the ticks are laid out on the X-axis'),
+            },
+          },
+        ],
+        [
+          {
+            name: 'number_format',
+            config: {
+              type: 'SelectControl',
+              freeForm: true,
+              label: t('Number format'),
+              renderTrigger: true,
+              default: 'SMART_NUMBER',
+              choices: D3_FORMAT_OPTIONS,
+              description: `${t('D3 format syntax: https://github.com/d3/d3-format')} ${t(
+                'Only applies when "Label Type" is set to show values.',
+              )}`,
+            },
+          },
+        ],
+      ],
+    },
+  ],
+  sectionOverrides: {
+    druidTimeSeries: {
+      controlSetRows: [['granularity', 'druid_time_origin'], ['time_range']],
+    },
+    sqlaTimeSeries: {
+      controlSetRows: [['granularity_sqla', 'time_grain_sqla'], ['time_range']],
+    },
+  },
+  controlOverrides: {
+    groupby: {
+      label: t('Series'),
+      validators: [validateNonEmpty],
+      clearable: false,
+      description: t('Categories to group by on the x-axis.'),
+    },
+    columns: {
+      label: t('Distribute across'),
+      multi: true,
+      description: t(
+        'Columns to calculate distribution across. Defaults to temporal column if left empty.',
+      ),
+    },
```

Comment:
Additional control to construct marginal distribution based on other than temporal dimension.